lib/storage/metadata/mongoclient/LogConsumer.js

```diff
@@ -150,15 +149,18 @@ class LogConsumer {
      * @return {undefined}
      */
     readRecords(params, cb) {
-        const recordStream = new ListRecordStream(this.logger);
         const limit = params.limit || 10000;
         const startIDandSeq = params.startSeq.toString().split('|');
         const startSeq = parseInt(startIDandSeq[0], 10) || 0;
-        lastEndID = startIDandSeq[1];
+        const lastEndID = startIDandSeq[1];
```

Comment:
should this change the `this.lastEndID`? just wondering, in the the old code, the global lastEndID is reassigned.

Reply:
They are in 2 separate classes, and the global was removed.